configKubernetes.md

```diff
@@ -234,19 +248,28 @@ Now that the pod is known to be running we need a way to find it.  Pods in kuber
 
 ```json
 {
-    "apiVersion": "v1beta1",
-    "containerPort": 80,
-    "id": "frontend",
+    "apiVersion": "v1beta3",
     "kind": "Service",
-    "labels": {
-        "name": "frontend"
+    "metadata": {
+        "name": "frontend-service",
+        "namespace": "default",
+        "labels": {
+            "name": "frontend"
+        }
     },
-    "port": 80,
-    "publicIPs": [
-        "NODE_PRIV_IP_1"
-    ],
-    "selector": {
-        "name": "apache"
+    "spec": {
+        "selector": {
+            "name": "apache"
+        },
+        "ports": [
+            {
+                "protocol": "TCP",
+                "port": 80
+            }
+        ],
+        "publicIPs": [
```

Comment:
deprecatedPublicIPs works?  I thought that was stopped.  In any case, I agree, should be made into a nodePort type v1 service.